### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by agh** [tube0013/tube_gateways#151] [Q&A] [answered]
Hey,

Firstly, thanks so much for what you're doing here, it makes getting up and running with a PoE-powered Zigbee Coordinator working with zigbee2mqtt so very easy. I'm looking to also get an equivalent for Bluetooth functional and want to modify your STL accordingly since the dimensions of that hardware are slightly different (it's a bit longer!), so could you please clarify as there's no LICENSE in the repository under what terms you are releasing all the things in the repository, including the 3D models?

Thanks <3

**Comment by tube0013** [comment, marked as answer]
I need some time to figure out licensing, but the stl's will be creative commons - [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/)

Let me know if you have any questions.
